modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/upgrade/DLServiceUpgrade.java

```diff
@@ -110,7 +109,7 @@ public void register(Registry registry) {
 				UpgradeDLFileEntryType());
 
 		registry.register(
-			"3.2.1", "3.2.2", new UpgradeDDMStructureLinkDLFileEntryType());
```

Comment:
@austinchiang I think that we can remove the version (downloading the bnd too) cause it is not published?

but I prefer to ask you before doing that,.